subscription/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseApiService.java

```diff
@@ -50,6 +51,9 @@ public DefaultSubscriptionBase createPlan(SubscriptionBuilder builder, Plan plan
     public List<DefaultSubscriptionBase> createPlans(Iterable<SubscriptionSpecifier> subscriptions, CallContext context)
```

Comment:
Since it's an internal API, can we remove this in favor of `createPlansWithAddOns`?